src/sasjs-request/index.js

```diff
@@ -74,13 +74,7 @@ export async function runSasJob(
     )
   }
 
-  const accessToken = await getAccessToken(
-    sasjs,
-    clientId,
-    clientSecret,
-    target,
-    isLocal
-  )
+  const accessToken = await getAccessToken(target)
```

Comment:
We don't need the bits above this where we're getting the `clientId` and `clientSecret` anymore.